### DISCUSSION TITLE
Support `sqlean-py` as a new SQLite driver

### DISCUSSION THREAD

**Discussion by edgarrmondragon** [sqlalchemy/sqlalchemy#10012] [Usage Questions] [answered]
[`sqlean.py`](https://github.com/nalgeon/sqlean.py) is fully DBAPI-compliant AFAICT since it claims to be a drop-in replacement to the stdlib's `sqlite3`, so it shouldn't be too difficult to add.

This would allow applications to use a more recent version of SQLite than is available by default on some machines without going through the hoops of upgrading the machine's installation, and also give users access to [a bunch of extensions](https://github.com/nalgeon/sqlean.py#extensions).

I'm happy to contribute this if there's interest from the maintainers 🙂.

**Comment by CaselIT** [comment, marked as answer]
hi

> claims to be a drop-in replacement to the stdlib's sqlite3

if that's so you can provide the dbapi to the dialect and it will use it. See https://docs.sqlalchemy.org/en/20/core/engines.html#sqlalchemy.create_engine.params.module

**Reply by edgarrmondragon** [reply]
Thanks, that works!
